modules/monitoring-knative-autoscaler-pods.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * serverless/monitoring-serverless.adoc
+
+[id="monitoring-knative-autoscaler-pods_{context}"]
+== Monitoring Knative Serving autoscaler pods using the {product-title} monitoring stack
+
+This section provides example  instructions for the visualization of Knative Serving pod autoscaling metrics by using the {product-title} monitoring tools.
+
+.Prerequisites
+
+* You must have the {product-title} monitoring stack installed. See the documentation about https://docs.openshift.com/container-platform/4.1/monitoring/cluster-monitoring/configuring-the-monitoring-stack.html#configuring-the-monitoring-stack[Configuring the monitoring stack].
+
+.Procedure
+
+. Navigate to the {product-title} Web console and authenticate.
+. Navigate to "Monitoring" → "Metrics".
```

Comment:
If these are specific entries in the web console surround them with `*` signs. For instance:

> Navigate to *Monitoring* -> *Metrics*.